base_local_planner/src/latched_stop_rotate_controller.cpp

```diff
@@ -221,10 +241,17 @@ bool LatchedStopRotateController::computeVelocityCommandsStopRotate(geometry_msg
     ROS_INFO("Goal position reached, stopping and turning in place");
     xy_tolerance_latch_ = true;
   }
+
   //check to see if the goal orientation has been reached
   double goal_th = tf::getYaw(goal_pose.getRotation());
   double angle = base_local_planner::getGoalOrientationAngleDifference(global_pose, goal_th);
-  if (fabs(angle) <= limits.yaw_goal_tolerance) {
+  if ((fabs(angle) <= limits.yaw_goal_tolerance ) || (latch_yaw_goal_tolerance_ && yaw_tolerance_latch_)) {
```

Comment:
latch_yaw_goal_tolerance_ && yaw_tolerance_latch_ is super confusing tbh

Reply:
yeah but it's the same naming scheme we used for the xy tolerances as well

Reply:
true, let's keep it